### ISSUE TITLE
guiset navbar none breaks FF68

### ISSUE THREAD

**Issue by diabolicfutility** [tridactyl/tridactyl#1728]
Tridactyl version: 1.16.1pre2946-cc77de0
Firefox version: Mozilla Firefox 68.0
Operating system: mac

---

I've had some broken behaviour in Firefox since updating to 68, and the cause of these issues is some Tridactyl-generated CSS in my userChrome.

The broken behaviour I've noticed is:

1. When opening Firefox, all non-pinned tabs don't load, and stay on blank pages.
2. Tridactyl's :tabopen can only open the new tab page.
3. Pinned tabs can alert the user by adding an indicator to the favicon. Normally this indicator disappears after visiting the tab, however when I switched to 68, there was no way to get rid of the indicator once it appeared.

The CSS that causes these issues is:

```
:root:not([customizing]) #nav-bar > :not(#customizationui-widget-panel) {
  display: none !important;
}
```

This CSS is added when using guiset navbar none, and possibly other guiset options.

**Comment by glacambre** [comment]
Yep, looks like they changed the way it works and they're going to do it again: https://hacks.mozilla.org/2019/07/firefox-68-bigints-contrast-checks-and-the-quantumbar/

**Comment by chrlsnchrg** [comment]
I tried filing a bug related to this, looks like it was shot down.

https://bugzilla.mozilla.org/show_bug.cgi?id=1566666

**Comment by bovine3dom** [comment]
Yeah. userChrome tweaks are unsupported and will probably be removed totally someday (presumably filing tickets might actually speed that up ;) ). I think we warn on `guiset` help. It's broken things a few times now.

**Comment by chrlsnchrg** [comment]
😞

**Comment by matthewdeancooper** [comment]
Currently:

```
guiset gui none
guiset tabs none
guiset navbar none
```

Have no effect. If i read correctly, these tweaks are no longer supported in firefox?

**Comment by bovine3dom** [comment]
`guiset gui none` should work, at least. If you're using Firefox ESR you may have the inverse of this issue: https://github.com/tridactyl/tridactyl/issues/2078

**Comment by krayon** [comment]
As mentioned in [Bugzilla Bug 1657294](https://bugzilla.mozilla.org/show_bug.cgi?id=1657294), the following is meant to work :
```css
#nav-bar{ height: 0px !important; min-height: 0px !important; visibility: hidden;}
```
I found that **with** `visibility: hidden;`, plugins that presented UI pop-ups or controls were not displaying, but without `visibility: hidden;` , apparently `[C-l]` and [C-k]` still "activate" it.

I'm using the later as I have addons that need to display things and the `^L`/`^K` thing doesn't bother me.

Tested in: v80.

**Comment by zeorin** [comment]
FWIW I'm very happy with the `hide_tabs_toolbars` and `autohide_toolbox` userChrome adjustments from https://github.com/MrOtherGuy/firefox-csshacks.
